versioned_hdf5/wrappers.py

```diff
@@ -34,6 +34,7 @@
 
 from .backend import DEFAULT_CHUNK_SIZE
 from .slicetools import build_data_dict
+from .staged_changes import StagedChangesArray
```

Comment:
While unused until the next PR, this line is necessary here because otherwise pytest would be the first to load the cythonized module and things would get weird and broken.